### PR TITLE
Fix geometry offset attribute for 2D and CV

### PR DESCRIPTION
**Pull request by hpinkos**
The offset vector needs to be projected for 2D and columbus view

Also fixed the doc for `OffsetGeometryInstanceAttribute`

Also, partial fix for #6365.  I wrapped the distance display condition 2D bounding sphere calculations in `if (!frameState.scene3DOnly)` so the example to longer crashes when `scene3DOnly: true`.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@hpinkos, thanks for the pull request! Maintainers, we have a signed CLA from @hpinkos, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by hpinkos** [comment]
Fixed the eslint error

File: Source/Scene/Primitive.js [hunk 1]
```diff
@@ -1394,17 +1414,75 @@ define([
...
             batchTable.setBatchedAttribute(i, radiusIndex, radius);
         }
 
         primitive._batchTableBoundingSpheresUpdated = true;
     }
 
+    var offsetScratchCartesian = new Cartesian3();
+    function updateBatchTableOffsets(primitive, frameState) {
+        var hasOffset = defined(primitive._batchTableAttributeIndices.offset);
+        if (!hasOffset|| primitive._batchTableOffsetsUpdated || frameState.scene3DOnly) {
```
Comment (bagnell) [inline Source/Scene/Primitive.js]: whitespace

File: Source/Scene/Primitive.js [hunk 2]
```diff
@@ -1394,17 +1414,75 @@ define([
...
+                continue;
+            }
+
+            var modelMatrix = primitive.modelMatrix;
+            if (defined(modelMatrix)) {
+                boundingSphere = BoundingSphere.transform(boundingSphere, modelMatrix, scratchBoundingSphere);
+            }
+
+            var center = boundingSphere.center;
+            center = ellipsoid.scaleToGeodeticSurface(center, center);
```
Comment (bagnell) [inline Source/Scene/Primitive.js]: You don't want to modify the bounding sphere right? Should this use a scratch variable?

**Comment by hpinkos** [comment]
Thanks @bagnell ! Updated

**Comment by bagnell** [comment]
👍

**Pull request merged**